### PR TITLE
Use ES6 syntax.

### PR DESCRIPTION
**Pull request by mattcollier**

### CODE REVIEW

File: generate-key-pair-modal-component.js [hunk 1]
```diff
@@ -5,27 +5,22 @@
  *
  * @author Dave Longley
  */
-define(['forge/js/pki'], function(pki) {
+import pki from 'forge/js/pki';
```
Comment (dlongley) [inline generate-key-pair-modal-component.js:8]: Does this work?

Comment (dlongley) [reply generate-key-pair-modal-component.js:8]: cc: @davidlehn

File: key-controller.js [hunk 2]
```diff
@@ -1,18 +1,15 @@
...
  * @author David I. Lehn
  * @author Dave Longley
  */
-define([], function() {
-
-'use strict';
-
 /* @ngInject */
-function factory($scope, brAlertService, brKeyService, brRefreshService) {
+export default function factory($scope, brAlertService, brKeyService,
```
Comment (dlongley) [inline key-controller.js]: Style nit: Should drop all params to a function when dropping any single one.

File: keys-component.js [hunk 3]
```diff
@@ -118,7 +111,8 @@ function Ctrl($scope, $routeParams, brAlertService, brKeyService,
         (identity.id === session.identity.id));
       if(self.isOwner ||
         (session.identity &&
-        hasPermission(session.identity, identity.id, 'PUBLIC_KEY_CREATE'))) {
+          hasPermission(session.identity, identity.id, 'PUBLIC_KEY_CREATE')
+        )) {
```
Comment (dlongley) [inline keys-component.js]: Bad indent?

**Review by dlongley — APPROVED** [review]

File: key-controller.js (hunk 2 above)
Comment (dlongley) [reply key-controller.js]: Adding another comment because I meant to request changes in the previous review and can't figure out how to edit it.

**Review by dlongley — CHANGES_REQUESTED** [review]

**Comment by mattcollier** [comment]
@dlongley other issues resolved, waiting to hear from @davidlehn on the forge question.

File: generate-key-pair-modal-component.js (hunk 1 above)
Comment (davidlehn) [reply generate-key-pair-modal-component.js:8]: Yes, the above (and extra line below) should work with forge 0.6.x.  0.7.x works differently and has some issues integrating with systemjs.  Eventually is will probably import all of forge and the extra line below will be removed.  And maybe in the far future forge will use es modules.

**Pull request merged**